packages/styles/scss/patterns/listsection/_listsection.scss

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection { 
+  .#{$prefix}--listsection {
+    color: $text-01;
+    background: $ui-background;
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
+    }
+    &__col {
+      @include carbon--make-col-offset(4, 16);
+      @include carbon--make-col(7, 16);
+    }
+    
+    &__row {
+      @include carbon--make-row;
+    }
+    &__title {
+      @include type-style('expressive-heading-05');
+    }
+    &__content {
+      @include type-style('body-long-02');
+    }
+    &__divider {
+      border-bottom: 1px solid $ui-04;
+    }
+    @include carbon--breakpoint('sm') {
+      &__col {
+        @include carbon--make-col-offset(0, 4);
+        @include carbon--make-col(4, 4);
+        padding-left: $layout-01;
+        padding-right: $layout-01;
+      }
+      &__title {
+        @include type-style('expressive-heading-05');
+        margin-top: $layout-05;
+        margin-bottom: $layout-03;
+      }
+      &__content {
+        @include type-style('body-long-02');
+      }
+      &__divider {
+        border-bottom: 1px solid $ui-04;
+        margin-bottom: $layout-01;
+        margin-top: $layout-06;
+        &__col {
+          @include carbon--make-col-offset(0, 4);
+          @include carbon--make-col(4, 4);
+        }
+      }
+    }
+    @include carbon--breakpoint('md') {
+      &__col {
+        @include carbon--make-col(7, 8);
+        padding-left: $layout-03;
+        padding-right: $layout-03;
+      }
+      &__listgroup { 
+        &__col {
+          @include carbon--make-col(7, 8);
+          padding-left: $layout-03;
+          padding-right: $layout-03;
+        }
+      }
+      &__divider {
+        &__col {
+          @include carbon--make-col(8, 8);
+          padding-left: $layout-01;
+          padding-right: $layout-01;
+        }
+      }
+    }
+    @include carbon--breakpoint('lg') {
+      &__col {
+        @include carbon--make-col-offset(4, 16);
+        @include carbon--make-col(7, 16);
+      }
+      &__title {
+        margin-top: $layout-06;
```

Comment:
According to the specs I think this was supposed to change in `md` breakpoint.

Reply:
vertical spacing between elements are same in sm & md, and increased at lg.
![Screen Shot 2019-10-29 at 3 13 26 PM](https://user-images.githubusercontent.com/8072521/67801006-bad67f00-fa5e-11e9-9b01-1d291a3e1662.png)
![Screen Shot 2019-10-29 at 3 13 40 PM](https://user-images.githubusercontent.com/8072521/67801011-bdd16f80-fa5e-11e9-82b3-b700c4f49b2f.png)